src/runmode-pfring.c

```diff
@@ -370,6 +373,44 @@ void *ParsePfringConfig(const char *iface)
         }
     }
 
+    if (ConfGetChildValueWithDefault(if_root, if_default, "copy-iface", &out_interface) == 1) {
+        if (strlen(out_interface) > 0) {
+            pfconf->out_interface = out_interface;
+        }
+    }
+
+    if (ConfGetChildValueWithDefault(if_root, if_default, "copy-mode", &copymodestr) == 1) {
+        if (pfconf->out_interface == NULL) {
+            SCLogInfo("Copy mode activated bu no destination"
```

Comment:
Yeah please do. AF_PACKET has the same issue, and I recently spent too much time debugging why it did not forward packets.